llvm/test/CodeGen/AArch64/GlobalISel/select-fold-lslfast.ll

```diff
@@ -12,17 +12,16 @@ declare void @foo()
 define i16 @halfword(ptr %ctx, i32 %xor72) nounwind {
```

Comment:
This test can be moved up a folder, if it's not just gisel. I would add the SDAG lines first too.

Can you remove the calls? It should still be possible to have different uses if it stores a different value.

Reply:
This was actually coped from `AArch64/aarch64-fold-lslfast.ll`. Let's add GISEL RUN lines to this file and delete the new file.

Reply:
The calls are originally there so I would prefer keeping them. Instead I added a new case `multi_use_half_word` to show multiple uses with stores (and without call).